README.md

```diff
@@ -1,2 +1,53 @@
-# tic-tac-toe
-Tic Tac Toe game app project
+# TIC TAC TOE
+
+In this project we build a Tic Tac Toe game app with Ruby. The game is played in the Terminal.
+
+## Game Description
+
+The Tic-tac-toe game is played by two people.
+Up on starting, the players enter their names and each gets assigned a symbol.
+The playground is a board consisting of a 3x3 grid with numbered cells
+
+The players take turns to enter  a number corresponding to the cell they want
+to mark. The goal is to mark three aligned cells with one's symbol. The alignment
+can be horizontal, vertical, or diagonal. The first player to achieve this goal is the 
+winner. When none of the players is able to align his three marks. The game ends in a draw 
+
+## Setup
+
```

Comment:
- [ ] README file explains the rules of the game. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone4/readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

Please, add a list that includes the rules of the game.